oak_runtime/src/channel.rs

```diff
@@ -76,14 +76,22 @@ pub struct Channel {
 
     /// The name for the channel.
     ///
-    /// The name does not have to be unique and can be empty. It is used in logs to help with
-    /// identifying channels during debugging.
-    pub name: String,
+    /// The name does not have to be unique and can be empty. In logs it is
+    /// combined with the id to form a unique debug_ig to identify channels.
+    name: String,
 
     /// Weak reference to the Runtime used for sending introspection events.
     runtime_weak: Weak<Runtime>,
 }
 
+impl Channel {
+    /// Returns a unique debug_id used to identify the node in the debug output,
+    /// consisting out of the provided [`ChannelId`], and the node's name.
+    fn get_debug_id(&self, channel_id: ChannelId) -> String {
```

Comment:
I'm confused, why is this both a method on Channel, but also expects a ChannelId to be passed in?

Reply:
We don't currently store the `id` in either the `Channel` or `Node` struct. Instead nodes/channnels are stored in a hashmap with the id as a key. 

Do you think I should add the `id` to the struct, eliminating the need to pass it for this method?

Reply:
`Channel` does have an `id`. So I have suggested to change `self.channel.get_debug_id(self.channel.id)`  to `self.channel.get_debug_id()`.

But we don't have a struct for `Node` (it is a trait), and `node_id` is assigned to a node when it is registered in the runtime.

Reply:
Makes sense! Overlooked that, will rm that argument for nodes. :)